.eslintrc.js

```diff
@@ -48,7 +48,9 @@ module.exports = {
             rules: {
                 // Disable these until we have converted the project to TS
                 '@typescript-eslint/explicit-module-boundary-types': 'off',
-                '@typescript-eslint/no-explicit-any': 'off'
+                '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
```suggestion
```

We must not allow any rules that can potentially get bad code in. Much better to ignore it for specific lines imo.
See comments for suggestions in how to deal with this.

Reply:
I don't think we can remove this until it's feasible to get a clean lint. There are lots of types that need to be set up and I don't want this PR to gets much bigger because there's already too much that can break...